pg_shard--1.1.sql

```diff
@@ -41,6 +41,12 @@ CREATE SCHEMA pgs_distribution_metadata
 	CREATE SEQUENCE shard_id_sequence MINVALUE 10000 NO CYCLE
```

Comment:
So I understand, in the SQL file if we detect CitusDB, we'll modify the view to insert using CitusDB's sequences (instead of these) right? I wanted to make sure we aren't potentially creating collisions here.